includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -71,9 +71,9 @@ private function getLangWithFallback( $lang ) {
 			'media-kit-contact' => null,
 			'social-facebook' => 'https://www.facebook.com/getfandom',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "social-facebook" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)